tests/admin_test.go

```diff
@@ -0,0 +1,94 @@
+			Expect(err).NotTo(HaveOccurred())
+			Eventually(sess).Should(Exit(0))
+			Expect(err).NotTo(HaveOccurred())
+			sess, err = start("deis perms:list --admin", adminTestData.Profile)
+			Eventually(sess).Should(Say("=== Administrators"))
+			Eventually(sess).Should(Say(adminTestData.Username))
+			Eventually(sess).Should(Say(testData.Username))
+			Expect(err).NotTo(HaveOccurred())
+			Eventually(sess).Should(Exit(0))
+			Expect(err).NotTo(HaveOccurred())
+			sess, err = start("deis perms:delete %s --admin", adminTestData.Profile, testData.Username)
+			Eventually(sess, defaultMaxTimeout).Should(Say("Removing %s from system administrators... done", testData.Username))
+			Expect(err).NotTo(HaveOccurred())
+			Eventually(sess).Should(Exit(0))
+			Expect(err).NotTo(HaveOccurred())
+			sess, err = start("deis perms:list --admin", adminTestData.Profile)
+			Eventually(sess).Should(Say("=== Administrators"))
+			Expect(sess).ShouldNot(Say(testData.Username))
+			Expect(err).NotTo(HaveOccurred())
+			Eventually(sess).Should(Exit(0))
+			Expect(err).NotTo(HaveOccurred())
+		})
+
+		It("can create, list, and delete app permissions", func() {
+			sess, err := start("deis perms:create %s --app=%s", adminTestData.Profile, testData.Username, testApp.Name)
+			Eventually(sess, defaultMaxTimeout).Should(Say("Adding %s to %s collaborators... done\n", testData.Username, testApp.Name))
+			Expect(err).NotTo(HaveOccurred())
+			Eventually(sess).Should(Exit(0))
+			Expect(err).NotTo(HaveOccurred())
+			sess, err = start("deis perms:list --app=%s", adminTestData.Profile, testApp.Name)
+			Eventually(sess).Should(Say("=== %s's Users", testApp.Name))
+			Eventually(sess).Should(Say("%s", testData.Username))
+			Expect(err).NotTo(HaveOccurred())
+			Eventually(sess).Should(Exit(0))
+			Expect(err).NotTo(HaveOccurred())
+			sess, err = start("deis perms:delete %s --app=%s", adminTestData.Profile, testData.Username, testApp.Name)
+			Eventually(sess, defaultMaxTimeout).Should(Say("Removing %s from %s collaborators... done", testData.Username, testApp.Name))
+			Eventually(sess).Should(Exit(0))
+			Expect(err).NotTo(HaveOccurred())
+			sess, err = start("deis perms:list --app=%s", adminTestData.Profile, testApp.Name)
+			Eventually(sess).Should(Say("=== %s's Users", testApp.Name))
+			Eventually(sess).ShouldNot(Say("%s", testData.Username))
+			Expect(err).NotTo(HaveOccurred())
+			Eventually(sess).Should(Exit(0))
+			Expect(err).NotTo(HaveOccurred())
+		})
+
+		It("regenerates the token for a specified user", func() {
+			sess, err := start("deis auth:regenerate -u %s", adminTestData.Profile, testData.Username)
+			Eventually(sess).Should(Say("Token Regenerated"))
+			Expect(err).NotTo(HaveOccurred())
+		})
+
+		// It("regenerates the token for all users", func() {
+		// 	sess, err := start("deis auth:regenerate --all", adminTestData.Profile)
+		// 	Eventually(sess).Should(Say("Token Regenerated"))
+		// 	Expect(err).NotTo(HaveOccurred())
+		// })
```

Comment:
k i marked it pending and added a comment as to why it was marked that way. I'll also open an issue